distributed/protocol/tests/test_rmm.py

```diff
@@ -0,0 +1,41 @@
+from distributed.protocol import serialize, deserialize
+import pickle
+import pytest
+
+numpy = pytest.importorskip("numpy")
+cuda = pytest.importorskip("numba.cuda")
+rmm = pytest.importorskip("rmm")
+
+
+@pytest.mark.parametrize("size", [0, 3, 10])
+def test_serialize_rmm_device_buffer(size):
+    DeviceBuffer = getattr(rmm, "DeviceBuffer", None)
+    if not DeviceBuffer:
+        pytest.skip("RMM pre-0.11.0 does not have DeviceBuffer")
+
+    x_np = numpy.arange(size, dtype="u1")
+    x = DeviceBuffer(size=size)
```

Comment:
I have a slight style preference to keep terms like `DeviceBuffer` namespaced and refer to it as `rmm.DeviceBuffer`.  I think that this is especially valuable when dealing with libraries that most Dask maintainers are unlikely to be familiar with.

Reply:
Sure that seems reasonable. Will update momentarily.

Reply:
Ok have pushed this change. Please take a look 🙂